air-quality-ui/src/components/header/ForecastBaseDatePicker.tsx

```diff
@@ -14,19 +14,29 @@ export const ForecastBaseDatePicker = (): JSX.Element => {
     },
   })
 
+  const IsTimeInvalid = (value: DateTime) => {
+    return value.minute != 0 || (value.hour != 0 && value.hour != 12)
```

Comment:
missing a superb moment to use remainder
```suggestion
    return value.minute != 0 || value.hour % 12 != 0
```

Reply:
You're quite right, thanks! 👍